### PR TITLE
feature-request: Added 'VPC Subnet Instances Present' plugin and test cases

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: plugins/aws/ec2/vpcSubnetInstancesPresent.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                });
+            }
+
+            describeSubnets.data.forEach(subnet => {
+                if(subnet.SubnetId && instanceSubnets[subnet.SubnetId]) {
+                    helpers.addResult(results, 0,
+                        `Subnet "${subnet.SubnetId}" has ${instanceSubnets[subnet.SubnetId]} instances attached`,
+                        region, subnet.SubnetArn);
+                } else {
+                    helpers.addResult(results, 2,
```
Comment (dipsubha06) [inline plugins/aws/ec2/vpcSubnetInstancesPresent.js:62]: this else can happen if somehow subnet.SubnetId is undefined ( which may be a rare case). In that case this addResult call also will fail as we are adding the subnet.SubnetId.

**Review by dipsubha06 — CHANGES_REQUESTED** [review]

File: plugins/aws/ec2/vpcSubnetInstancesPresent.js (hunk 1 above)
Comment (giorod3) [reply plugins/aws/ec2/vpcSubnetInstancesPresent.js:62]: @AkhtarAmir I agree with Subhadip.

File: plugins/aws/ec2/vpcSubnetInstancesPresent.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                        `Subnet "${subnet.SubnetId}" does not have any instance attached`,
+                        region, subnet.SubnetArn);
```
Comment (giorod3) [inline plugins/aws/ec2/vpcSubnetInstancesPresent.js:64]:
```suggestion
                    helpers.addResult(results, 2,
                        `Subnet does not have any instance attached`,
                        region, subnet.SubnetArn);
```

Comment (AkhtarAmir) [reply plugins/aws/ec2/vpcSubnetInstancesPresent.js]: Fixed

**Review by giorod3 — APPROVED** [review]

**Pull request merged**